### PR TITLE
Added Webhooks Fixes #4

### LINKED ISSUE #4: Add Webhook capability
Many APIs such as Twilio require webhooks for calllbacks.

[More Information](https://www.twilio.com/platform/webhooks) can be found on the Twilio.

### PR DESCRIPTION
**Pull request by TheThirdOne**

### CODE REVIEW

**Comment by TheThirdOne** [comment]
This should be merged after #14

File: baton/modules/echo/echo.go [hunk 1]
```diff
@@ -9,7 +16,18 @@ func (e Echo) RunCommand(cmd string, body interface{}, resp chan<- interface{})
...
-}
\ No newline at end of file
+}
+
+func (e Echo) Handler() *mux.Router {
+	m := mux.NewRouter()
+	m.PathPrefix("/").HandlerFunc(echo)
+	return m
+}
+func echo(w http.ResponseWriter, r *http.Request) {
```
Comment (zachlatta) [inline baton/modules/echo/echo.go:29]: Should there be an empty line between these two functions? Was `gofmt` run on this file?

File: baton/modules/neutrino/neutrino.go [hunk 2]
```diff
@@ -1,55 +1,61 @@
...
 	if err := json.NewDecoder(data.Body).Decode(&out); err != nil {
 		log.Println(err)
 		return nil
 	}
 	resp <- out
 	return nil
-}
\ No newline at end of file
+}
+func (n Neutrino) Handler() *mux.Router {
```
Comment (zachlatta) [inline baton/modules/neutrino/neutrino.go:60]: Should there be an empty line between this and the previous function?

File: cmd/maestro/maestro.go [hunk 3]
```diff
@@ -89,9 +89,8 @@ The options are:
 	go baton.Run()
 	m := http.NewServeMux()
 	m.Handle("/baton/", http.StripPrefix("/baton", baton.Handler()))
+	m.Handle("/webhooks/", http.StripPrefix("/webhooks", baton.WebhookHandler()))
```
Comment (zachlatta) [inline cmd/maestro/maestro.go]: Might it make sense to register this in https://github.com/hackedu/maestro/blob/master/baton/handler.go#L12?

File: baton/modules/echo/echo.go (hunk 1 above)
Comment (TheThirdOne) [inline baton/modules/echo/echo.go:29]: It was run on the file, but `gofmt` doesn't edit newlines after functions.

File: cmd/maestro/maestro.go (hunk 3 above)
Comment (TheThirdOne) [inline cmd/maestro/maestro.go]: To clarify, would that webhook for Echo be something like `/baton/webhooks/Echo/`?

Comment (zachlatta) [inline cmd/maestro/maestro.go]: Yeah.

Comment (TheThirdOne) [inline cmd/maestro/maestro.go]: ok, sure

**Pull request merged**